data/sources.json

```diff
@@ -3,8 +3,7 @@
     "id": 430,
     "name": "Online shop",
     "items_ids": [
-      309,
-      255
+
```

Comment:
Please make sure this information is in this file, and stays there when the program loads and closes